IDE/WoCCode.md

```diff
@@ -0,0 +1,132 @@
+# Title
+
+WoCCode: Peer mentoring for the Women of Color coding community
+
+## Summary 
+
+Women of Color (WoC) -- which includes Black, indigenous, and Latinx
+minorities -- are the most underrepresented demographic from high
+paying industries, including software engineering. Yet WoC are flush
+with the talent, creativity, and social insight to solve complex
+problems and lead in the software industry. This program seeks to
+unite WoC and other members of minoritized groups, to battle feelings
+of isolation and marginalization that are prevalent in educational
+institutions and industry settings. Using a dedicated Slack space and
+cohorts of 4-5 people, we will create a peer mentoring network for WoC
+and members of other minoritized groups to engage, receive feedback,
+and troubleshoot issues around their participation in open source
+software communities, including Astropy.
+
+## Team 
+
+Program Lead: Lia Corrales (@eblur)
+
+Program Mentors and Slack moderators:
+* Lia Corrales
+* Kelle Cruz
+* TBD (invited Camille Avestruz)
+
+Anticipated 2020 cohort: 20 people from underrepresented groups in the
+United States (Black, indigenous, disabled, and gender-minority
+coders) or other underrepresented groups from the national stage (see
+Plan). We are open to any career stage, from undergraduate to research
+professionals.
+
+Guest webinar speakers: 6 industry or research professionals
+
+## Plan
+
+We will solicit applications to join the program via Twitter, AAS
+CSMA, and by emails to academic department chairs. We will accept
+applications from anyone, and will select a cohort in a way that
+prioritizes most marginalized demographics.
+
+We operate under the principle that supporting the most marginalized
+in our communities supports the well-being of all members of the
+Astropy community. While the Astropy project has shown considerable
+diversity in its international demographics, the majority of non-US
+participants are still overwhelmingly European (white), male, and from
+privileged classes. This program seeks to engage with people from
+groups that are marginalized in the United States, the Global South
+(Central America, Latin America, Africa, and less-privileged classes
+of Southeast Asia), or Global North (European, Northern Asia,
+Australian) members from marginalized demographics such as being
+indigenous, non-binary or queer, and disabled.
+
+We aim for 20 participants, to be separated into cohorts of 4-5 people
+with similar interests.
+
+Participants will be connected via Slack, with one channel for each
+cohort and a general chat channel. Slack moderators will be assigned
+to moderate cohort chat spaces to help answer technical questions
+regarding code. Each cohort is expected to hold regular meetings, 2-4
+times per month, for peer-mentoring. (These sessions should be
+independent of Slack and will not include the Slack moderator assigned
+to them.)
```

Comment:
I really like the idea of a dedicated Slack channel, and this seems like something that would be easy-enough to set up in general, so long as there are people willing to take the time to moderate it and grow its membership.  
I know people who have started women-only online spaces (e.g. Facebook groups) for their industries (and I believe I have heard of WoC-specific spaces as well but obviously I'm not in them so I don't know any details).  These have proven extremely valuable for peer-mentoring, networking, etc.